tests/suites/0_stateless/20+_others/20_0001_show.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
why not use sqllogictest?

Reply:
Try to use sqllogictest but it can not success. the result is ok, but databend-sqllogictest can not compare the diff.

Reply:
>  but databend-sqllogictest can not compare the diff.

same in my other PR. seems like a bug in sqllogictest. Anyway, we should not add new test in the legacy test framework...

Reply:
Ok, I got it. https://docs.rs/sqllogictest/0.17.2/sqllogictest/runner/fn.default_validator.html function normalize_string trim \n as " "